packagedb/api.py

```diff
@@ -695,17 +695,21 @@ def get_resolved_purls(packages):
     for items in packages or []:
         purl = items.get('purl')
         vers = items.get('vers')
-        
+
         try:
             parsed_purl = PackageURL.from_string(purl)
         except ValueError:
-            unsupported_purls.add(purl)
+            unsupported_purls.add(purl) if purl else None
```

Comment:
```suggestion
            unsupported_purls.add(purl)
```

Reply:
IMO if purl is `None` then we should not add it to the list of unsupported_purls list.

Reply:
@keshav-space That is covered by the first suggestion, where we continue if `purl` is None.

Reply:
ack, i missed that :)

Reply:
No worries! My apologies for not elaborating on that in my initial suggestion